lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -36,6 +36,7 @@ function dosomething_global_init() {
   $url_variables = dosomething_global_get_url_variables();
   $user_variables = dosomething_global_get_user_variables();
   $node_variables = dosomething_global_get_node_variables();
+  dosomething_helpers_getcampaign_run($node_variables['nid']);
```

Comment:
actually, you have an underscore in the function def. So how is the above one working?